packages/pegasus/src/pegasus.js

```diff
@@ -459,9 +459,10 @@ const makePegasus = (zcf, board, namesByAddress) => {
      * @param {ERef<Peg>} pegP the peg over which to transfer
      * @param {DepositAddress} depositAddress the remote receiver's address
      * @param {string} [memo] the memo to attach to ics transfer packet
+     * @param {string} [sender] the sender to attach to ics transfer packet
```

Comment:
Seeing as the protocol may be extended further in the future, and we haven't yet deployed Pegasus to a chain, I'd prefer to create an options bag now.

Would you be able to implement this by replacing the existing `memo = ''` argument with `opts`?  If not, I can probably carve out some time to have a go at it by next week.

```js
/**
 * @typedef {object} SenderOptions Additional sender options
 * @property {string} [memo] the memo to attach to ics transfer packet
 * @property {string} [sender] the sender to attach to ics transfer packet
 */
``` 

```suggestion
     * @param {SenderOptions} [opts] additional sender options
```